src/app-layer-dns-udp-rust.c

```diff
@@ -65,6 +65,21 @@ static uint16_t DNSUDPProbe(Flow *f, uint8_t direction,
     return ALPROTO_DNS;
 }
 
+static uint16_t DNSUDPProbeSrv(Flow *f, uint8_t direction,
```

Comment:
This will no longer be necessary once a version of #4513 is merged.

Reply:
This is still a necessary bug to fix, only this bug fix will need to be moved to Rust once #4604 is merged